pandas/core/generic.py

```diff
@@ -1652,6 +1652,55 @@ def _repr_latex_(self):
     strings before writing.
     """
 
+    _shared_docs['to_pickle'] = """
+    Pickle (serialize) %(klass)s object to input file path.
+
+    Parameters
+    ----------
+    path : string
+        File path where the pickled %(klass)s object will be stored.
+    compression : {'infer', 'gzip', 'bz2', 'xz', None}, default 'infer'
+        A string representing the compression to use in the output file.
+
+        .. versionadded:: 0.20.0
+    protocol : int
+        Int which indicates which protocol should be used by the pickler,
+        default HIGHEST_PROTOCOL (see [1], paragraph 12.1.2). The possible
+        values for this parameter depend on the version of Python. For
+        Python 2.x, possible values are 0, 1, 2. For Python>=3.0, 3 is a
+        valid value. For Python >= 3.4, 4 is a valid value.A negative value
+        for the protocol parameter is equivalent to setting its value to
+        HIGHEST_PROTOCOL.
+
+        .. [1] https://docs.python.org/3/library/pickle.html
+        .. versionadded:: 0.21.0
+
+    Examples
+    --------
+    >>> original_df = pd.DataFrame({"foo": range(5), "bar": range(5, 10)})
+    >>> original_df
+       foo  bar
+    0    0    5
+    1    1    6
+    2    2    7
+    3    3    8
+    4    4    9
+    >>> original_df.to_pickle("./dummy.pkl")
+
+    >>> unpickled_df = pd.read_pickle("./dummy.pkl")
+    >>> unpickled_df
+       foo  bar
+    0    0    5
+    1    1    6
+    2    2    7
+    3    3    8
+    4    4    9
+
+    See Also
+    --------
+    pandas.read_pickle
```

Comment:
can you add similar to what you adding in other See Also below

Reply:
added.